blockchain/blocks/src/tipset.rs

```diff
@@ -107,6 +107,10 @@ impl Tipset {
     pub fn min_ticket(&self) -> Ticket {
         self.first_block().ticket().clone()
     }
+    /// Returns the block with the smallest ticket of all blocks in the tipset
+    pub fn min_ticket_block(&self) -> &BlockHeader {
+        &self.first_block()
+    }
```

Comment:
If all this does is expose `first_block`, maybe get rid of `first_block` altogether? This is the better name anyway

Reply:
oops yeah I missed that, no need for both

Reply:
Yeah, I kinda just left it there before i wasn't sure if they person who put it in intents to use it for something. Ill Remove first_block

Reply:
I put it there during a refactor because it was used several times within the file

Reply:
ok i removed first_block in favour of min_ticket_block